src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1408,16 +1408,18 @@ public function resolveEnvPlaceholders($value, $format = null, array &$usedEnvs
         }
         $envPlaceholders = $bag instanceof EnvPlaceholderParameterBag ? $bag->getEnvPlaceholders() : $this->envPlaceholders;
 
+        $completed = false;
         foreach ($envPlaceholders as $env => $placeholders) {
             foreach ($placeholders as $placeholder) {
-                if (false !== stripos($value, $placeholder)) {
+                if (\is_string($value) && false !== stripos($value, $placeholder)) {
```

Comment:
Is the added check useful? I feel like the added break is enough, isn't it?

Reply:
Yeah, the break should be enough. I included the `is_string()` check to protect the `stripos()` call, but it shouldn't be necessary. I can remove it if that's preferred.

Reply:
yes, let's remove the check if might make ppl spend time wondering when this can be false, when it cannot :)